integrations/nlu-snips/src/SnipsNlu.ts

```diff
@@ -26,6 +25,9 @@ export class SnipsNlu extends NluPlugin<SnipsNluConfig> {
 
   getDefaultConfig(): SnipsNluConfig {
     return {
+      input: {
+        supportedTypes: [],
```

Comment:
Not sure if this is necessary here as it's already part of the default config of `NluPlugin` https://github.com/jovotech/jovo-framework/blob/8b093d9db8cef19a0dcc2dc9fc541980fc60911a/framework/src/NluPlugin.ts#L25